jujugui/static/gui/src/app/components/entity-details/content/content.js

```diff
@@ -224,6 +293,46 @@ YUI.add('entity-content', function() {
     },
 
     /**
+      Generate the list of terms if available.
+
+      @method _generateTerms
+      @return {Array} The terms markup.
+    */
+    _generateTerms: function() {
+      const terms = this.state.terms;
+      let content;
+      if (this.state.termsLoading) {
+        content = (
+          <juju.components.Spinner />);
+      } else if (terms.length === 0) {
+        return null;
+      } else {
+        const items = terms.map(item => {
+          const url = [
+            this.props.staticURL, 'terms', item.name, item.revision].join('/');
+          return (
+            <li key={item.name}>
+              <a className="link"
+                href={url}
```

Comment:
Don't we have a component to show terms directly in the GUI? It is used in the deployment flow. Mayeb we could reuse that, rather than opening the direct link?